cmake/compiler_flags_Intel_Fortran.cmake

```diff
@@ -1,40 +1,44 @@
-# (C) Copyright 2020-2020 NOAA/NWS/NCEP/EMC
+# (C) Copyright 2009-2016 ECMWF.
```

Comment:
@srherbener and @PatNichols what should we put here?

Reply:
I think the copyright notices look good now. Thanks for fixing those!